.github/workflows/build.yml

```diff
@@ -1,8 +1,6 @@
 name: "Build"
 on:
   push:
-    tags:
```

Comment:
I found it helpful to run the action builds for any commit, but only publish snaps on release.

Reply:
I would prefer if this was left unchanged.

Reply:
In that case `edge` snaps won't be built from commits to master, so I can remove all the `set-grade` stuff and assume they're only built off tagged releases.

Is that your intent? I'm not sure why one wouldn't want CI builds for every commit, but I'll defer to your preference.